package.json

```diff
@@ -7,14 +7,24 @@
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
+    "lint:fix": "eslint --ext \".js,.vue\" --ignore-path .gitignore . --fix",
     "lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",
     "lint": "npm run lint:js",
-    "test": "jest"
+    "precommit": "run-s pretty:staged lint:js",
+    "prepare": "husky install",
+    "pretty:staged": "pretty-quick --staged",
+    "pretty": "pretty-quick",
+    "test": "jest",
+    "test:watch": "jest --watch"
   },
   "dependencies": {
     "@nuxtjs/axios": "5.13.6",
     "@nuxtjs/pwa": "3.3.5",
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/user-event": "^13.2.0",
+    "@testing-library/vue": "^5.8.1",
     "core-js": "3.15.1",
+    "husky": "^7.0.1",
     "nuxt": "2.15.7"
```

Comment:
My IDE yelled at me because Vue isn't in the dependency list... we should probably add it ^^"